api/misc.inc

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Would it be better to give this file a different name since there is already a /pinc/misc.inc. Or even a name reflecting what it does.

Reply:
Hmm. Naming is hard, and all that. The problem is, I just needed somewhere to put a function that's used by two files and I don't know what else might end up in there. `misc.inc` is terribly vague, I'll give you that. How about `common.inc`?

Reply:
Other name brainstorming: `api_misc.inc`? `api_common.inc`? (yes, they are already in `api/` so it's a bit redundant)

Reply:
Either of these sound reasonable to me. Even if it's redundant, it can be helpful if all you have to go by is the filename, instead of the full path.

Reply:
`api_common.inc` it is...